### PR TITLE
Front page update

### PR DESCRIPTION
**Pull request by kmdalal**
### Summary ###

This is a fairly small edit that mainly cleans up the front page of the user guide.

- Clarifies the Plexil releases that are covered.
- Replaces the non-working Gmail-based support email with the previous Sourceforge email and removes mention of any Plexil discussion emails.  
- Updates the documentation build instructions to account for the `docs/.nojekyll` file

Only `.rst` files and `README.md` were edited.

### CODE REVIEW

**Review by plexil-bh — APPROVED** [review]

File: GettingStarted/Introduction.rst [hunk 1]
```diff
@@ -10,12 +10,12 @@ included in the software distribution. The latest version of this manual
...
-better documented, so any feedback you have is greatly appreciated.
-Comments and questions are welcome. If you encounter any material you
-find confusing, or is inadequate for your needs, please contact
-plexildiscuss@gmail.com and we'll do our best to help you and to
-improve the manual.
+We are striving to make the language and system increasingly usable
+and better documented, so any feedback you have is greatly
+appreciated.  Comments and questions are welcome. If you encounter any
+material you find confusing, or is inadequate for your needs, please
+contact plexil-support@lists.sourceforge.net and we'll do our best to
```
Comment (plexilcrf) [inline GettingStarted/Introduction.rst]: Suggest you change the list address to plexil-support (at) groups (dot) io.

File: PLEXILLanguage/PLEXILOverview.rst [hunk 2]
```diff
@@ -278,15 +278,13 @@ given in the chapter titled :ref:`PLEXIL, XML, and Emacs <XMLSchemaEmacs>`.
...
--  *plexil-support* email will be sent only to the |PLEXIL| developers.
-
-Please feel free to add yourself to *plexil-discussion* using the
-`mailing lists <plexildiscuss@gmail.com>`_
+If you have problems installing or using PLEXIL, find bugs in it, or
+have feature requests, please open an issue at
+https://github.com/plexil-group/plexil/issues.
 
+If you have questions about PLEXIL, please contact the support team
+at plexil-support@lists.sourceforge.net.
```
Comment (plexilcrf) [inline PLEXILLanguage/PLEXILOverview.rst]: Please update the mailing list address.

File: index.rst [hunk 3]
```diff
@@ -1,11 +1,17 @@
-This is the main page for the PLEXIL wiki. For PLEXIL's main project
-page, which has downloads and other information, go here:
-https://github.com/plexil-group/plexil
+This is the main page of the PLEXIL User's Guide. It documents PLEXIL
+Release 4.6, which is the version of PLEXIL found on its `main` Git
```
Comment (plexilcrf) [inline index.rst]: The 'main' branch only exists on Babelfish. On GitHub, it's releases/plexil-4.6

File: index.rst [hunk 4]
```diff
@@ -54,8 +60,7 @@ more information on the background and applications of |PLEXIL|.
 See the sidebar (column to the left of this page) for links to many
 |PLEXIL| topics.
 
-If you have questions, please email us at
-plexildiscuss@gmail.com
+If you have questions, please email us at plexil-support@lists.sourceforge.net.
```
Comment (plexilcrf) [inline index.rst]: Please update mailing list address.

File: GettingStarted/Introduction.rst (hunk 1 above)
Comment (kmdalal) [reply GettingStarted/Introduction.rst]: Done.

File: PLEXILLanguage/PLEXILOverview.rst (hunk 2 above)
Comment (kmdalal) [reply PLEXILLanguage/PLEXILOverview.rst]: Done.

File: index.rst (hunk 3 above)
Comment (kmdalal) [reply index.rst]: Fixed.

File: index.rst (hunk 4 above)
Comment (kmdalal) [reply index.rst]: Done.

**Review by plexilcrf — APPROVED** [review]

**Pull request merged**